ext/native-decls/GetHeliTailRotorHealth.md

```diff
@@ -8,9 +9,10 @@ apiset: server
 float GET_HELI_TAIL_ROTOR_HEALTH(Vehicle vehicle);
 ```
 
+**Note**: This native is deprecated, please use `GET_HELI_REAR_ROTOR_HEALTH` instead.
```

Comment:
```suggestion
**Note**: This native is deprecated, please use [`GET_HELI_REAR_ROTOR_HEALTH`](#_0x33EE6E2B) instead.
```

Reply:
Oh i was searching how to do it, i don't understand what is 0x33EE6E2B? it's the joaat of the native?

Reply:
Yes all Cfx native will be a jooat.